### ISSUE TITLE
Add HDR?

### ISSUE THREAD

**Issue by IShadowDukeI** [Lyall/WukongTweak#10]
From engine.ini. This will enable HDR support, only set by ini. So people can use this with MWK - SPF

[SystemSettings]
r.allowhdr=1
r.HDR.EnableHDROutput=1
r.HDR.Display.OutputDevice=5
r.HDR.Display.ColorGamut=2
r.HDR.UI.CompositeMode=1
r.FullScreenMode=0
r.HDR.UI.Level=1.5
r.HDR.Display.MidLuminance=54
r.HDR.Display.MaxLuminance=750

**Comment by Lyall** [comment]
If it can be enabled (and stays enabled) via Engine.ini then it's probably not something I'd add. The main benefit of setting these other CVars in code is that we can override ones that are changed by the game script/code.

**Comment by Foul-Tarnished** [comment]
"MWK - SPF" mod is pure snake oil

**Comment by Lyall** [comment]
> "MWK - SPF" mod is pure snake oil

Is this mod https://www.nexusmods.com/blackmythwukong/mods/13 ?

**Comment by Foul-Tarnished** [comment]
> > "MWK - SPF" mod is pure snake oil
> 
> Is it this mod https://www.nexusmods.com/blackmythwukong/mods/13 ?

Yes

**Comment by IShadowDukeI** [comment]
> "BMWK - SPF" mod is pure snake oil

Yeah, figures. :D i only saw that with the prev. addition of the RT shadows, so i thought i mention it. ;)

**Comment by IShadowDukeI** [comment]
> If it can be enabled (and stays enabled) via Engine.ini then it's probably not something I'd add. The main benefit of setting these other CVars in code is that we can override ones that are changed by the game script/code.

Right.
And yes, it stays enabled. Could cause shadow flickering in some scenes, also depending on your settings. But since i like HDR a lot, i like the benefit.

**Comment by Foul-Tarnished** [comment]
BTW you can always use SpecialK to inject HDR.

It's usually really good (not fake HDR) as it inject into the engine, and
is highly customizable.

Le lun. 26 août 2024, 22:12, IShadowDukeI ***@***.***> a
écrit :

**Comment by IShadowDukeI** [comment]
I know, not a fan tho. There is "native" HDR in Wukong, its just not enabled, probably because of these shadow glitches, maybe its coming properly with a patch.

**Comment by ZeppLu** [comment]
Is it possible to make the dark areas lighter on SDR screen? The contrast is really annoying
